app/src/main/java/com/hedvig/app/util/extensions/ActivityExtensions.kt

```diff
@@ -133,6 +133,11 @@ private fun Activity.openAppSettings() {
     startActivity(intent)
 }
 
+fun Activity.openEmail(title: String) {
+    val intent = Intent(Intent.ACTION_SEND)
```

Comment:
How does this work? 🤔

Reply:
Good catch, this needed more testing. Fixed now